.babelrc

```diff
@@ -5,9 +5,15 @@
       {
         "modules": false
       }
-    ]
+    ],
+    "@babel/preset-typescript"
   ],
   "plugins": [
-    ["@babel/plugin-transform-runtime", { "regenerator": true }]
+    [
+      "@babel/plugin-transform-runtime",
+      {
+        "regenerator": true
+      }
+    ]
   ]
-}
+}
```

Comment:
Your editor might need a config touch up to insert proper line breaks at the end of files.